test-requirements.in

```diff
@@ -4,3 +4,5 @@ pyright
 pytest
 pytest-operator
 ruff
+ops-scenario
+cryptography
```

Comment:
They are in the tests requirements because they are needed by the tests themselves, we create certificates in tests code. Yes technically we could only set it in the `requirements.in` because it's already used in source code but since it is needed by both I think it makes more sense to have it in both.